client/src/components/Overview/AddToCart 2.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect} from 'react';
```

Comment:
This file is called AddToCart 2. It looks like it is my AddToCart component so I'm wondering why it is called AddToCart 2 and if it is a duplicate file.

Reply:
Dont know how that happened. Just deleted it.